homeassistant/components/device_automation/__init__.py

```diff
@@ -174,6 +174,13 @@ async def _async_get_device_automations(
                 device_results, InvalidDeviceAutomationConfig
             ):
                 continue
+            if isinstance(device_results, Exception):
```

Comment:
Is `return_exceptions=True` no longer required for this?

Reply:
We need `return_exceptions` to be `True` for the multi-device case so we can use the ones that are working.

We need `return_exceptions` to be `False` for the single device case since we know we can raise given it won't affect fetching the other devices